pandas/core/format.py

```diff
@@ -207,7 +207,11 @@ def _get_formatted_index(self):
         return fmt_index, have_header
 
     def _get_formatted_values(self):
-        return format_array(self.tr_series.get_values(), None,
```

Comment:
I think I changed this in another PR I am doing, use do

``return format_array(self.tr_series.values, ...)`` and this wil work

``format_array`` already handles this type of dispatching